src/components/FeedbackWidget.tsx

```diff
@@ -17,157 +16,63 @@ import { translateMessageId } from "../utils/translations"
-  box-shadow: 0px 4px 4px ${({ theme }) => theme.colors.tableItemBoxShadow};
-  position: sticky;
-  bottom: 1rem;
-  margin-left: auto;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    bottom: ${({ bottomOffset }) => 1 + bottomOffset}rem;
-    margin-top: 150vh;
-  }
-  right: 1rem;
-  z-index: 98; /* Below the mobile menu */
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  &:hover {
-    cursor: pointer;
-    transform: scale(1.1);
-    transition: transform 0.2s ease-in-out;
-  }
-  transition: transform 0.2s ease-in-out;
-`
-
-const ModalBackground = styled.div`
-  display: block;
-  position: fixed;
-  inset: 0;
-  background-color: rgba(0, 0, 0, 0.2);
-  z-index: 1001; /* Above the nav bar */
-`
-
-const Container = styled.div<{
+interface FixedDotProps extends ButtonProps {
   bottomOffset: number
-}>`
-  display: flex;
-  box-sizing: border-box;
-  width: 300px;
-  background-color: ${({ theme }) => theme.colors.ednBackground};
-  border: 1px solid ${({ theme }) => theme.colors.buttonColor};
-  box-shadow: 0px 4px 4px ${({ theme }) => theme.colors.tableItemBoxShadow};
-  border-radius: 0.25rem;
-  position: fixed;
-  right: 2rem;
-  bottom: 5rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    offset-position: bottom -300px;
-    bottom: ${({ bottomOffset }) => 5 + bottomOffset}rem;
-  }
-  z-index: 1002; /* Above the ModalBackground */
-
-  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
-    width: auto;
-    left: 1rem;
-    right: 1rem;
-  }
-
-  &:hover {
-    transform: scale(1.02);
-    transition: transform 0.2s ease-in-out;
-  }
-  transition: transform 0.2s ease-in-out;
-
-  flex-direction: column;
-  align-items: center;
-  text-align: center;
-  padding: 2rem;
-
-  .title {
-    font-weight: 700;
-    font-size: 1.25rem;
-    line-height: 1.5rem;
-  }
-
-  .subtitle {
-    font-weight: 400;
-    font-size: 1rem;
-    line-height: 1.25rem;
-  }
-
-  .timing {
-    font-weight: 700;
-    font-size: 0.75rem;
-    line-height: 1rem;
-    letter-spacing: 0.025em;
-    color: ${({ theme }) => theme.colors.searchBorder};
-  }
-
-  a {
-    width: 100%;
-    color: ${({ theme }) => theme.colors.text} !important;
-  }
-`
-
-const ButtonContainer = styled.div`
-  display: flex;
-  flex-wrap: nowrap;
-  gap: 1.25rem;
-  width: 100%;
-  * {
-    flex: 1;
-    color: ${({ theme }) => theme.colors.white};
-    font-weight: 700;
-  }
-`
-
-const StyledFeedbackGlyph = styled(FeedbackGlyph)`
-  path {
-    fill: ${({ theme }) => theme.colors.white};
-  }
-`
-
-const IconContainer = styled(NakedButton)`
-  position: absolute;
-  right: 0.5rem;
-  top: 0.5rem;
-  cursor: pointer;
-  &:hover {
-    transform: scale(1.1);
-    transition: transform 0.2s ease-in-out;
-  }
-  transition: transform 0.2s ease-in-out;
-`
-
-export interface IProps {
-  className?: string
+}
+const FixedDot: React.FC<FixedDotProps> = ({
```

Comment:
Just want to mention that there is a Chakra component called `IconButton` https://chakra-ui.com/docs/components/icon-button that perhaps could be a replacement for this. Not saying that this is in the scope of this PR but perhaps it is a better solution in the long term.

Reply:
Yes this component has more changes to come in a separate PR, thanks for noting!